namelists/echam/6.3.04p1/cmip6/namelist.echam

```diff
@@ -0,0 +1,139 @@
+&parctl
+	nproca=24
+	nprocb=48
+/
+&runctl
+	NPROMA=72
```

Comment:
Careful here: this might have influence on the comparability of two runs if the number is different. From the handbook, this controls: "vector length of calculations in grid point space"

 https://www.cen.uni-hamburg.de/en/icdc/data/atmosphere/docs-atmo/echam6-userguide.pdf

Reply:
I agree with this, in all the namelist I've seen, also the ones of @chrisdane, `nproma: 8`. @antoniofis10000, can you check that for Tido's simulation NPROMA is 72? 

I would suggest to set NPROMA to 8 by default. Then you can add to your runscript if Tido's simulations had that:
```yaml
echam:
     add_namelist_changes:
         namelist.echam:
            runctl:
                nproma: 72
```